public/index.html

```diff
@@ -36,6 +36,8 @@
     -->
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
 
+    <link rel="preconnect" href="%REACT_APP_AMPLITUDE_PROXY_URL%" />
```

Comment:
preconnect is new to me--maybe we should add one for infura?

Reply:
same here, i'm just following a suggestion from the original PR #6554

Reply:
we shouldn't add too many, and the infura subdomain may not be known at first (the subdomains would each require a distinct connection).

I think it's best to just update this one. right now it includes a path, which is unnecessary; it also needs to be crossorigin. See https://www.debugbear.com/resource-hint-validator?url=app.corn-staging.com.